tests/integration/express-server.spec.ts

```diff
@@ -1246,6 +1246,27 @@ describe('Express Server', () => {
     });
   });
 
+  describe('Sub resource', () => {
```

Comment:
Copy this to the base hapi and koa server.

Reply:
I'm not sure if I correctly understand this comment. Do you mean I should copy-paste the exact same code to the other two tests (hapi-server and koa-server)? Or is there a shared place to copy this to so I don't need to copy-paste the same test three times? I did the former but if you meant the latter I might need a bit more guidance.